### PR TITLE
fix: fill the missing type in tracingpolicy example

### PR DESCRIPTION
**Pull request by sadath-12**
fills the missing ```type``` value under the ```args``` for tracingpolicy example

### CODE REVIEW

**Review by lambdanis — COMMENTED** [review]
I don't have full context on why this directory exists, but I'm wondering if these should be moved under the top level `examples/tracingpolicy`.

**Comment by lambdanis** [comment]
> I don't have full context on why this directory exists, but I'm wondering if these should be moved under the top level `examples/tracingpolicy`.

ok, so both directories are used in tests:
* `pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml` is embedded [here](https://github.com/cilium/tetragon/blob/ef1c5f5db1c0f7721dd8e93829036e1f5162d943/pkg/tracingpolicy/generictracingpolicy_test.go#L397)
* `examples/tracingpolicy` is walked [here](https://github.com/cilium/tetragon/blob/main/pkg/tracingpolicy/generictracingpolicy_test.go#L470)

The former was initially added for tests, and only then the latter started being used in tests too. I think we should move `tracepoint-lseek-pid.yaml` to `examples/tracingpolicy` and update the test, but check if there is a similar TracingPolicy there first. Could you do it @sadath-12?

**Comment by sadath-12** [comment]
Sure thanks @lambdanis

**Comment by sadath-12** [comment]
@lambdanis  the ```//go:embed`` there I think uses ```go list``` to list files in the current dir and that might be the reason the yaml is there .

**Comment by sadath-12** [comment]
> tracepoint-lseek-test.yaml

yes it can be removed since ```pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml``` kind of similar

File: pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ spec:
     event: "sys_enter_lseek"
     args:
     - index: 7
+      type: "auto"
```
Comment (lambdanis) [inline pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml:11]: "auto" is the default, so this is technically not needed, but doesn't hurt either

**Review by lambdanis — APPROVED** [review]

File: pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml (hunk 1 above)
Comment (sadath-12) [reply pkg/tracingpolicy/examples/tracepoint-lseek-pid.yaml:11]: got it 👍😊

**Comment by lambdanis** [comment]
The CI failure is a known issue: #2010. It's unrelated, so I'm merging this PR.

**Pull request merged**